queue/index.php

```diff
@@ -1,12 +1,131 @@
+<title>Queue | VITA Lincoln | University of Nebraska&ndash;Lincoln</title>
+<!-- TemplateEndEditable -->
+<!-- TemplateBeginEditable name="head" -->
+<link rel="stylesheet" href="/dist/queue/queue.css">
+<link rel="stylesheet" href="https://ajax.googleapis.com/ajax/libs/angular_material/1.1.4/angular-material.min.css">
+<!-- TemplateEndEditable -->
+<!-- TemplateParam name="class" type="text" value="" -->
+</head>
+<body class="@@(_document['class'])@@" data-version="4.1">
+	<?php wdnInclude("/wdn/templates_4.1/includes/skipnav.html"); ?>
+	<div id="wdn_wrapper">
+		<input type="checkbox" id="wdn_menu_toggle" value="Show navigation menu" class="wdn-content-slide wdn-input-driver" />
+		<?php wdnInclude("/wdn/templates_4.1/includes/noscript-padding.html"); ?>
+		<header id="header" role="banner" class="wdn-content-slide wdn-band">
+			<div id="wdn_header_top">
+				<span id="wdn_institution_title"><a href="http://www.unl.edu/">University of Nebraska&ndash;Lincoln</a></span>
+				<div id="wdn_resources">
+					<?php wdnInclude("/wdn/templates_4.1/includes/wdnResources.html"); ?>
+					<?php wdnInclude("/wdn/templates_4.1/includes/idm.html"); ?>
+					<?php wdnInclude("/wdn/templates_4.1/includes/search.html"); ?>
+				</div>
+			</div>
+			<div id="wdn_logo_lockup">
+				<div class="wdn-inner-wrapper">
+					<?php wdnInclude("/wdn/templates_4.1/includes/logo.html"); ?>
+						<span id="wdn_site_affiliation"><!-- TemplateBeginEditable name="affiliation" --><!-- TemplateEndEditable --></span>
+						<span id="wdn_site_title"><!-- TemplateBeginEditable name="titlegraphic" -->VITA Lincoln<!-- TemplateEndEditable --></span>
+				</div>
+			</div>
+		</header>
+		<div id="wdn_navigation_bar" class="wdn-band">
+			<nav id="breadcrumbs" class="wdn-inner-wrapper" role="navigation" aria-label="breadcrumbs">
+				<!-- TemplateBeginEditable name="breadcrumbs" -->
+				<ul>
+					<li><a href="http://www.unl.edu/" title="University of Nebraska&ndash;Lincoln" class="wdn-icon-home">UNL</a></li>
+					<li><a href="/" title="VITA Lincoln">VITA Lincoln</a></li>
+					<li>Queue</li>
+				</ul>
+				<!-- TemplateEndEditable -->
+			</nav>
+			<div id="wdn_navigation_wrapper">
+				<nav id="navigation" role="navigation" aria-label="main navigation">
+					<!-- TemplateBeginEditable name="navlinks" -->
+					<?php include "$root/sharedcode/navigation.php"; ?>
+					<!-- TemplateEndEditable -->
+					<?php wdnInclude("/wdn/templates_4.1/includes/navigation-addons.html"); ?>
+				</nav>
+			</div>
+		</div>
+		<div class="wdn-menu-trigger wdn-content-slide">
+			<label for="wdn_menu_toggle" class="wdn-icon-menu">Menu</label>
+			<?php wdnInclude("/wdn/templates_4.1/includes/share.html"); ?>
+		</div>
+		<main id="wdn_content_wrapper" role="main" class="wdn-content-slide" tabindex="-1">
+			<div id="maincontent" class="wdn-main">
+				<div id="pagetitle">
+					<!-- TemplateBeginEditable name="pagetitle" -->
+					<h1>Queue</h1>
```

Comment:
Having it say "Queue" here looks kinda odd... should we remove this? Although it would ruin the consistency

Reply:
I think it would be fine to remove just the line containing ```<h1>Queue</h1>```.

Reply:
Done.